tests/test_sensor.py

```
@@ -1418,6 +1418,53 @@ def test_monotonic_clock_lag_reset_after_date_flip(mock_plant, freezer):
     assert _read(entity, mock_plant, 0.3) == 0.3
 
 
+def test_monotonic_deferred_reset_admitted_after_gap(mock_plant, freezer):
+    """The deferred owed-reset call honours the elapsed-scaled band too: after a
+    skewed (negative) boundary reading defers the decision, a reset that lands a
+    couple of hours later — above the 0.5 kWh floor but within 15 kW × elapsed —
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)